src/View/layout.html.twig

```diff
@@ -5,19 +5,28 @@
     <meta http-equiv="x-ua-compatible" content="ie=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="icon" href="/assets/images/favicon.png">
+    <link rel="shortcut icon" href="assets/img/favicon.png" type="image/x-icon">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/remixicon/3.4.0/remixicon.css" crossorigin="">
+
 
     <title>{% block title %}{{ name }}{% endblock %}</title>
 
     <link rel="stylesheet" href="/assets/css/style.css">
+  
     <link rel="stylesheet" href="/assets/css/footer.css">
     <link href="https://fonts.googleapis.com/css2?family=Playpen+Sans:wght@500;900&amp;display=swap" rel="stylesheet">
-    
+
+   
     {% block stylesheet %}{% endblock %}
 </head>
 
 <body>
+    {% include 'Item/components/header.html.twig' %}
```

Comment:
If you do this every page will have the image of your hero section.

Reply:
yes the header must be in every page no !

Reply:
Your header is include the discipline. You also want to view that in page Detail Formation and page Contact???

Reply:
yes because that's we have dans la maquette ! @assma8jaballah